### PR TITLE
Adding boolean to tell topological navigation not to care for orientation...

### PR DESCRIPTION
**Pull request by Jailander**
... in the final node

Necessary for Walking group as discussed with @marc-hanheide and @cdondrup

### CODE REVIEW

**Comment by marc-hanheide** [comment]
We discussed the side effects of this in depth, but @hawesie @bfalacerda @cburbridge and other people at G4S should be made aware of this and object if need be. This PR changes the ROS msg `GotoNode` in topo nav. It adds a boolean flag, making the msg incompatible with running/not updated code that used that message. I believe this not be a problem, because:
- The default behaviour is that the bool is false, and this is backward compatible, so if any existing code uses the msg and does not set the new boolean value, it behaves as before.
- Once this is released, we will automatically build all depending packages on jenkins and an `apt-get update` will update it all correctly
- [Searching for `GotoNodeGoal` in all STRANDS repos](https://github.com/search?l=&q=GotoNodeGoal+user%3Astrands-project&ref=advsearch&type=Code&utf8=%E2%9C%93) showed it's only being used in AAF-related repos and in [`sm_base_executor.py`](https://github.com/strands-project/strands_executive/blob/hydro-release/task_executor/src/task_executor/sm_base_executor.py), but in the latter only when running in `TOPOLOGICAL_NAV` mode.

But please be aware that one should as a precaution restart the whole system when updating. I'm :+1: for merging this, as we need it for AAF.

File: topological_navigation/scripts/navigation.py [hunk 1]
```diff
@@ -150,6 +151,8 @@ def executeCallback(self, goal):
         if self.monit_nav_cli :
             self.cancelled = False
             self.preempted = False
+            self.no_orientation = goal.no_orientation
+            print "NO ORIENTATION (%s)" %self.no_orientation
```
Comment (marc-hanheide) [inline topological_navigation/scripts/navigation.py:155]: although, @Jailander, I really want to urge you to _not_ use `print` but `rospy.loginfo` or similar (for reasons that should be obvious)

**Comment by bfalacerda** [comment]
yeah, we dont use the topological nav action so it should be fine

**Comment by hawesie** [comment]
If @bfalacerda passes it then I'm happy.

**Comment by Jailander** [comment]
Yes sorry my pr was not very descriptive, but I have tested it with former code and it is backwards compatible so it shouldn't affect anyone

**Pull request merged**